zappnode-front/src/landing/components/navigation/navigate.js

```diff
@@ -1,30 +1,36 @@
 import React from 'react';
 import '../css/tailwind.generated.css';
 import '../css/landing.css';
```

Comment:
Styles should be imported in App.js or Index.js

Reply:
have you addressed the above-request change? is it good to merge?